src/get-section.c

```diff
@@ -64,7 +64,7 @@ int main(int argc, char ** argv) {
         ABORT(255);
     }
 
-    singularity_message(DEBUG, "Iterating through file looking for sections matching: \%%s\n", section);
+    singularity_message(DEBUG, "Iterating through file looking for sections matching: %s\n", section);
```

Comment:
Can I ask what the reasoning behind removing "%" is? It would make sense to include the % sign in the string so it reads out "%pre" or "%setup". I'm not sure if % is the correct way to escape a % sign, but I know that %% works. Maybe use that instead?